src/main/java/labs/introtoprogramming/lab5/object/Builder/POJO/FacePlate.java

```diff
@@ -0,0 +1,54 @@
+package labs.introtoprogramming.lab5.object.Builder.POJO;
+
+import java.util.ArrayList;
+
+public class FacePlate {
+
+    public ArrayList<VertexFace> vertices = new ArrayList<VertexFace>();
+    public Material material = null;
+    public Material map = null;
+
+    public FacePlate() {
+    }
+
+    public void add(VertexFace vertex) {
+        vertices.add(vertex);
+    }
+    public VertexNormal faceNormal = new VertexNormal(0, 0, 0);
+
+    public void calculateTriangleNormal() {
```

Comment:
I think this function is redundant here. Face can be made of more than 3 vertices. Or if any FacePlate is made only of 3 vertices than rewrite this using cross product. And please use counterclockwise conversion to reduce misunderstandings.